### DISCUSSION TITLE
[Solved] Nothing does work anymore

### DISCUSSION THREAD

**Discussion by Profi420** [libre-tube/LibreTube#5288] [General]
I can not start playing Videos anymore. Has someone any idea how i can fix this issue?
I have the newest Version. Android 12

**Comment by Vicvi86** [comment]
I have the same problem, I tried to add another instance but it still doesn't work.

**Comment by Silther** [comment]
Enable HLS until it is fixed

**Reply by Profi420** [reply]
What is HLS? Were can i enable it?

**Reply by begin-theadventure** [reply]
> Were can i enable it?

Settings -> Audio and video -> Use HLS
